src/models/event.js

```diff
@@ -1143,6 +1144,41 @@ utils.extend(MatrixEvent.prototype, {
     getTxnId() {
         return this._txnId;
     },
+
+    /**
+     * Get a copy/snapshot of this event. The returned copy will be loosely linked
+     * back to this instance, though will have "frozen" event information. Other
+     * properties may mutate depending on the state of this instance at the time
```

Comment:
Hmm, what properties are you thinking of...?

Reply:
`sender`, mostly. Will clarify